src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -173,6 +173,17 @@ def __writeSub(self, executable, nJobs, location, processors):
 
         executable = os.path.join(self.workingDirectory, executable)
 
+        useCredentials = ""
```

Comment:
Should be `useCredentials = "use_x509userproxy = true"` as otherwise if there isn't a token the pilot will fail with "no proxy found".

Reply:
Yes, fixed